CTRE/MotorControl/SensorCollection.cs

```diff
@@ -10,6 +11,8 @@ public class SensorCollection
         /** Low level object. */
         MotController_LowLevel _ll;
 
+        //SensorBiases _biases;
```

Comment:
@rgreenblatt @JCaporuscio remove this comment

Reply:
Fixed in https://github.com/CrossTheRoadElec/Phoenix-netmf-lib/commit/686a734bee597cc231b42dd35ff8864a19ef651b